clients/privacy-center/constants/index.ts

```diff
@@ -1,9 +1,34 @@
 /* eslint-disable import/prefer-default-export */
-
-import { Config, IdentityInputs } from "~/types/config";
 import configJson from "~/config/config.json";
+import {
+  isV1ConsentConfig,
+  translateV1ConfigToV2,
+} from "~/features/consent/helpers";
+import {
+  Config,
+  IdentityInputs,
+  V1Consent,
+  V2Config,
+  V2Consent,
+} from "~/types/config";
+
+/**
+ * Transform the config to the latest version so that components can
+ * reference config variables uniformly.
+ */
+const transformConfig = (config: Config): V2Config => {
+  if (isV1ConsentConfig(config.consent)) {
+    const v1ConsentConfig: V1Consent = config.consent;
+    const translatedConsent: V2Consent = translateV1ConfigToV2({
+      v1ConsentConfig,
+    });
+    const temp: V2Config = { ...config, consent: translatedConsent };
+    return temp;
```

Comment:
probably want to remove the `temp` var and just condense to `return { ...config, consent: translatedConsent }`

Reply:
IIRC we needed this for the typechecker?